test-clients/devGenesisKeypair.pem

```diff
@@ -0,0 +1,7 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
+MIHDMH8GCSqGSIb3DQEFDTByMFEGCSqGSIb3DQEFDDBEBDDvfX2byw3gj5mIXVwq
+yPIChKHpVaxsCPiqIcPpmZo4X2DGuNKJNlIsA/31eVYdatMCAicQMAwGCCqGSIb3
+DQIKBQAwHQYJYIZIAWUDBAEqBBDS2zOxNai/TEZZmOpuVj1QBECIGDOePb9HdPqn
+uwu3exrZhV1qrqdxj1OWFfQBJtTQTvWlJCUkFPlfoizq8o90McHVtyx5RMUEwYL+
+2G/INvmB
+-----END ENCRYPTED PRIVATE KEY-----
```

Comment:
Is this pem copied from `hedera-node/data/onboard/devGenesisKeypair.pem`? One minor suggestion, we should put in under `test-clients/src/main/resource` instead.